elections/2021-TOC2/election_desc.md

```diff
@@ -0,0 +1,39 @@
+# 2021 Voters Guide - Knative TOC Special Election
+
+This special election has 2 goals:
+- **Runoff election** for the remaining seat due for election this year on the 
+  Knative Technical Oversight Committee. This runoff is between 
+  David Protasowski and Scott Nichols. The one with the most votes in this special
+  election will serve a two (2) year term.
+- **Filling the vacancy** to replace Grant Rodgers who recently stepped down and serve the remainder
+  of his term ending during the 2022 election. This vacancy could not be filled using 
+  candidates from the previous election because after filling the above
+  seat, VMware and Red Hat / IBM will be at the 
+  [maximum number of seats](https://github.com/knative/community/blob/main/mechanics/TOC.md) (2) 
+  held by employees from the same organization (or conglomerate, in the case of companies owning
+  each other). This means that we need to have at least one (preferably more) nominee who does not 
+  work at Red Hat, IBM, or VMware. 
+
```

Comment:
Add:

Please rank all candidates together according to your preference and evaluation of them as potential TOC members, regardless of company affiliation.  The candidates for the two seats will be selected by the SC, according to the total preference rankings and TOC eligibility rules.

... as in, we don't need them to try to rank people based on company affiliation or anything.  We'll do that.